apps/projects/app/script.js

```diff
@@ -296,15 +308,27 @@ function loadRequestsData({repoId, issueNumber}) {
 
 function getRequest(repoId, issueNumber, applicantId) {
   return new Promise(resolve => {
-    app.call('getApplicant', repoId, issueNumber, applicantId).subscribe( (address) => {
-      app.call('getAssignmentRequest', repoId, issueNumber, address).subscribe( (hash) => {
-        let contentJSON
-        ipfs.get(hash, (err, files) => {
-          for(const file of files) {
-            contentJSON = JSON.parse(file.content.toString('utf8'))
-          }
-          resolve({contributorAddr: address, ...contentJSON})
-        })
+    app.call('getApplicant', repoId, issueNumber, applicantId).subscribe( ({address, hash}) => {
```

Comment:
I believe the subscription should be passed an object from a call the returns more than one parameter

Reply:
Yeah this is correct, can we change the variable names to the return names from getApplicant for clarity or do you think this is more clear? i.e. applicant application